src/Icon.js

```diff
@@ -33,20 +33,26 @@ const getPath = ({ name, legacy }) => {
   return icons.legacy[name] || icons[name] || aliases[name]
 }
 
+// Remove `space` props from the `svg` element prevents react warnings
+const CleanSvg = cleanElement('svg')
+CleanSvg.propTypes = {
+  ...propTypes.space
+}
+
 const Base = ({ name, size, legacy, ...props }) => {
   const icon = getPath({ name, legacy })
   if (!icon) return false
 
   return (
-    <svg
+    <CleanSvg
```

Comment:
Is it possible that any apps had margins assigned to their `Icon`'s?  I wouldn't want to break anyone's layout.

Reply:
The wrapper just remove the mr, mx, etc.. props. that are not used from the svg element. The warning in the console is thrown because there are some app (e.g. search) that add this values.
...but while I'm writing this I've seen that probably is enough to remove the prop from the parent component. I'll check.